Orgs: Add future deprecation notice

torkelo opened the pull request:
* [ ] Refine wording
* [ ] Create an issue to link to for feedback. The issue should have a little bit more detail but messaging should basically say the same thing as the infobox. 
* [ ] Add deprecation info to changelog

![Screenshot from 2020-05-11 11-37-02](https://user-images.githubusercontent.com/10999/81547164-c397fe00-937b-11ea-9c7f-5cdc3d59cb26.png)

torkelo committed:
Orgs: Fist draft of orgs deprecation infobox

bergquist commented on public/app/features/admin/AdminListOrgsPage.tsx:
```diff
@@ -29,6 +29,22 @@ export const AdminListOrgsPages: FC = () => {
           <div className="page-action-bar">
+            <InfoBox branded>
+              <VerticalGroup spacing="xs">
+                <Badge text="Info" color="blue" icon="info-circle" />
```
```suggestion
                <Badge text="Warning" color="blue" icon="info-circle" />
```
Im not sure if warnings exists but I think the "level" should be higher.

torkelo replied:
removed the badge, the warnings and red color look too strong

bergquist commented on public/app/features/admin/AdminListOrgsPage.tsx:
```diff
@@ -29,6 +29,22 @@ export const AdminListOrgsPages: FC = () => {
+                  users would have a better experience using Teams instead. As such, we are considering de-emphasizing
+                  and eventually deprecating Organizations in future Grafana releases. If you would like to provide
+                  feedback or describe your use case, please do so{' '}
+                  <a className="external-link" href="https://github.com/grafana/grafana/issues/xxx">
```
Could you please create this issue now? Would be nice to have a real issue to link to so we don't forget to update it :)

bergquist replied:
The issue can be named placeholder for now since it doesn't affect the link.

torkelo replied:
added issue link

bergquist reviewed (commented):
:+1:

dprokop commented:
@torkelo not a fan of using the branded info box for this purpose. I was thinking about it as a way of bringing playful attention to new features we want to highlight.

For deprecation warnings I think we should use something more "serious" and toned down visually. Maybe @jessover9000 can help? Or just use regular info box

bergquist committed:
adds warning about potentially deprecating orgs (#24512)

torkelo commented:
But this is supposed to be a bit friendly so I think this box looks fine as, do not like the old infobox, this is so much better

ryantxu commented on pkg/services/sqlstore/sqlstore.go:
```diff
@@ -101,13 +101,42 @@ func (ss *SqlStore) Init() error {
+
+		if resp[0].Count > 1 {
+			ss.log.Warn(`[Deprecation notice]`)
+			ss.log.Warn(`Organizations is a feature that only 1% of Grafana users are utilizing, and we feel that most of those`)
```
```suggestion
			ss.log.Warn(`Organizations is a feature that only 1% of Grafana users are using, and we feel that most of those`)
```

ryantxu commented:
The sentance is full of "use, user, utilize, using" can we try some alternatives?

torkelo committed:
Updates

torkelo commented:
Updated text and added issue link, https://github.com/grafana/grafana/issues/24588
![Screenshot from 2020-05-12 18-37-25](https://user-images.githubusercontent.com/10999/81721109-a5b3c180-947f-11ea-8dbc-d70a8bcbcaa5.png)

torkelo committed:
Fixed ts issue
